scheduler/src/cook/mesos/task.clj

```diff
@@ -85,6 +85,9 @@
         environment (cond-> (assoc (util/job-ent->env job-ent)
                               "COOK_JOB_UUID" (-> job-ent :job/uuid str))
                             group-uuid (assoc "COOK_JOB_GROUP_UUID" (str group-uuid))
+                            (:cpus resources) (assoc "COOK_JOB_CPUS" (-> resources :cpus int str))
+                            (:gpus resources) (assoc "COOK_JOB_GPUS" (-> resources :gpus int str))
+                            (:mem resources) (assoc "COOK_JOB_MEM_MB" (-> resources :mem int str))
```

Comment:
Why are we converting to `int` here? `:cpus` and `:mem` can both be floating point